src/main/java/se/sundsvall/installation/api/InstallationResource.java

```diff
@@ -20,5 +38,18 @@ class InstallationResource {
 		this.installationService = installationService;
 	}
 
+	@Operation(summary = "Get uninstalled installations",
+		description = "Resource returns all uninstalled installations matching provided search parameters")
+	@ApiResponses(value = {
+		@ApiResponse(responseCode = "200", description = "Successful operation", useReturnTypeSchema = true),
+		@ApiResponse(responseCode = "400", description = "Bad request", content = @Content(mediaType = APPLICATION_PROBLEM_JSON_VALUE, schema = @Schema(oneOf = {Problem.class, ConstraintViolationProblem.class}))),
+		@ApiResponse(responseCode = "500", description = "Internal Server error", content = @Content(mediaType = APPLICATION_PROBLEM_JSON_VALUE, schema = @Schema(implementation = Problem.class))),
+		@ApiResponse(responseCode = "502", description = "Bad Gateway", content = @Content(mediaType = APPLICATION_PROBLEM_JSON_VALUE, schema = @Schema(implementation = Problem.class)))
+	})
+	@GetMapping(produces = {APPLICATION_JSON_VALUE, APPLICATION_PROBLEM_JSON_VALUE})
+	public ResponseEntity<InstallationDetailsResponse> getInstallations(
```

Comment:
Use own class for response from service, do not let DWR "bleed trough" (InstallationDetailsResponse).